smarts/core/utils/tests/fixtures.py

```diff
@@ -199,7 +199,6 @@ def large_observation():
         ),
         drivable_area_grid_map=DrivableAreaGridMap(
             metadata=GridMapMetadata(
-                created_at=1649853761,
                 resolution=0.1953125,
```

Comment:
I had to remove `created_at` because it made the `GridMapMetadata` object non-deterministic.